packages/web/docs/mdx-components.js

```diff
@@ -0,0 +1,6 @@
+import { useMDXComponents as getDocsMDXComponents } from '@theguild/components/server';
```

Comment:
I fixed `mdx-components.ts/tsx` issues, you can move docs components directly in this file

Reply:
Note: This turned out to cause another problem, we discussed it in DMs, and I reverted. We'll change to `mdx-components.tsx` later.